src/app/components/cesium/map/map.js

```diff
@@ -193,7 +196,7 @@ class CesiumComponent extends Component {
   addTick () {
     const { viewer } = this.state
     if (this.ticking) return
-    viewer.clock.onTick.addEventListener(c => this.onTick(c))
+    viewer.clock.onTick.addEventListener(this.onTick)
```

Comment:
Have you tested this? I remember this was here because the context got lost somehow...